src/DI/ElasticsearchExtension.php

```diff
@@ -5,25 +5,40 @@
 use Elasticsearch\Client;
 use Elasticsearch\ClientBuilder;
 use Nette\DI\CompilerExtension;
+use Nette\Schema\Expect;
+use Nette\Schema\Schema;
+use stdClass;
 
+/**
+ * @property-read stdClass $config
+ */
 class ElasticsearchExtension extends CompilerExtension
 {
 
-	/** @var mixed[] */
-	private $defaults = [
-		'hosts' => ['localhost'],
-	];
+	public function getConfigSchema(): Schema
+	{
+		return Expect::structure([
+			'hosts' => Expect::anyOf(Expect::arrayOf('string'), Expect::structure([
```

Comment:
Is it possible to use
```yaml
hosts
  - localhost
  - host: 192.168.0.1
    port: 1234
```
or could it be just array of strings or array of arrays?